release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3313,6 +3313,43 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.14.22
+[id="ocp-4-14-22"]
+=== RHSA-2024:1891 - {product-title} 4.14.22 bug fix update and security update
+
+Issued: 2024-04-25
+
+{product-title} release 4.14.22, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:1891[RHSA-2024:1891] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1897[RHSA-2024:1897] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.14.22 --pullspecs
+----
+
+[id="ocp-4-14-22-enhancements"]
+==== Enhancements
+
+[id="ocp-4-14-22-validate-control-plane-replicas"]
+===== Number of configured control plane replicas validated
+
+* Previously, the number of control plane replicas could be set to an invalid value, such as `2`. With this release, a validation is added to prevent any misconfiguration of the control plane replicas at the ISO generation time. (link:https://issues.redhat.com/browse/OCPBUGS-31885[*OCPBUGS-31885*])
```

Comment:
```suggestion
* Previously, you could set the number of control plane replicas to an invalid value, such as `2`. With this release, a validation is added to prevent any misconfiguration of the control plane replicas at the ISO generation time. (link:https://issues.redhat.com/browse/OCPBUGS-31885[*OCPBUGS-31885*])
```